module/VuFind/src/VuFind/Auth/PasswordAccess.php

```diff
@@ -77,13 +77,20 @@ public function getConfig()
     public function authenticate($request)
     {
         $config = $this->getConfig()->toArray();
-        $req_password = trim($request->getPost()->get('password', ''));
+        $requestPassword = trim($request->getPost()->get('password', ''));
+        foreach ($config['PasswordAccess']['access_user_hashed'] ?? [] as $username => $passwordHash) {
+            if (
+                password_verify($requestPassword, $passwordHash)
+            ) {
```

Comment:
I don't think this needs to take up three lines:

```suggestion
            if (password_verify($requestPassword, $passwordHash)) {
```